example/main.go

```diff
@@ -12,7 +12,6 @@ import (
 	"fortio.org/cli"
 	"fortio.org/log"
 	"fortio.org/terminal"
-	"golang.org/x/term"
```

Comment:
@ccoVeille re https://github.com/fortio/terminal/pull/1#discussion_r1709985875 your comment made me do this PR as the error was bleeding through, not anymore